packages/backend/src/dbt/profiles.ts

```diff
@@ -49,6 +50,10 @@ const credentialsTarget = (
                     threads: credentials.threads,
                     keepalives_idle: credentials.keepalivesIdle,
                     sslmode: credentials.sslmode,
+                    sslrootcert: path.resolve(
+                        __dirname,
+                        '../services/warehouseClients/amazon-trust-ca-bundle.crt',
```

Comment:
@ZeRego should we move this logic into the `lightdashConfig` service?

Reply:
Not seeing a reason why we should 🤔  Because we intent to make the path to the redshift certificate configurable ? If so, I would rather have a file input in the UI when they are setting up the redshift credentials.

Reply:
Just thinking of centralising all fixed configuration for the application. So all constants that define Lightdash on startup can be found in one place